compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/draganddrop/DragAndDrop.desktop.kt

```diff
@@ -40,6 +40,13 @@ actual class DragAndDropTransferData(
     @property:ExperimentalComposeUiApi
     val supportedActions: Iterable<DragAndDropTransferAction>,
 
+    /**
+     * The initial transfer action when the drag starts; must be one of the values in
+     * [supportedActions].
+     */
+    @property:ExperimentalComposeUiApi
+    val initialAction: DragAndDropTransferAction,
```

Comment:
Can we add some default value?

Reply:
I thought about it, but
1. There's no obvious default value.
2. It needs to match the supportedActions, and it could be confusing.